Resources/Private/JavaScript/Host/Service/TYPO3CR/NodeTreeService.js

```diff
@@ -88,7 +91,11 @@ class NodeTreeService {
                 })
             })
             .then(response => response.json())
-            .then(json => this.store.dispatch(actions.UI.PageTree.setSubTree(storePath, json)));
+            .then(json => {
+                const nodeWhichIsNotLoadingAnymore = $set(updatedRootInStore, 'isLoading', false);
+                this.updateNode(nodeWhichIsNotLoadingAnymore);
+                this.store.dispatch(actions.UI.PageTree.setSubTree(storePath, json));
+            });
```

Comment:
@grebaldi has written what I was about to write, and what I had already suggested to Tyll. Great we're on the same page here!